contracts/pToken.sol

```diff
@@ -15,6 +15,8 @@ contract PToken is
     bytes32 public constant MINTER_ROLE = keccak256("MINTER_ROLE");
     bytes4 public ORIGIN_CHAIN_ID;
 
+    event HookFailed();
```

Comment:
Since we aren't using "hooks" anymore, we can reify this name perhaps?

```suggestion
    event ReceiveUserDataFxnFailed();
```

or something?

Reply:
Renamed to `ReceiveUserDataFailed`